### PR TITLE
Add unit test for ProjectionUtils public method

### LINKED ISSUE #27317: Add unit test for ProjectionUtils public method
This issue is to add unit tests for the ProjectionUtils class, it includes the following subtasks:

- [ ]  ProjectionUtils#getColumnLabelFromAlias
- [ ]  ProjectionUtils#getColumnNameFromFunction

You can refer this pr - https://github.com/apache/shardingsphere/pull/25386 to add unit test.

### PR DESCRIPTION
**Pull request by pandalee99**
Fixes #27317.
add a test class ProjectionUtilsTest
and two test  method:
  - ProjectionUtilsTest#getColumnLabelFromAlias
  - ProjectionUtilsTest#getColumnNameFromFunction

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [x] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Comment by pandalee99** [comment]
> The operation was canceled.

strange.

File: infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.apache.shardingsphere.infra.database.postgresql.PostgreSQLDatabaseType;
+import org.apache.shardingsphere.infra.database.spi.DatabaseType;
+import org.apache.shardingsphere.infra.util.quote.QuoteCharacter;
+import org.apache.shardingsphere.sql.parser.sql.common.value.identifier.IdentifierValue;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class ProjectionUtilsTest {
```
Comment (strongduanmu) [inline infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java:32]: Please remove public modifier for ProjectionUtilsTest.

File: infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    
+    @Test
+    void assertGetColumnLabelFromAlias() {
+        IdentifierValue alias = null;
+        DatabaseType databaseType = null;
+        alias = new IdentifierValue("Data", QuoteCharacter.NONE);
+        assertThat(ProjectionUtils.getColumnLabelFromAlias(alias, databaseType), is(alias.getValue()));
```
Comment (strongduanmu) [inline infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java:39]: Can you remove alias and new IdentifierValue here directly?

File: infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        alias = new IdentifierValue("Data");
+        databaseType = new PostgreSQLDatabaseType();
+        assertThat(ProjectionUtils.getColumnLabelFromAlias(alias, databaseType), is(alias.getValue().toLowerCase()));
```
Comment (strongduanmu) [inline infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java:42]: Can you change expected string to constant?

File: infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+public class ProjectionUtilsTest {
+    
+    @Test
+    void assertGetColumnLabelFromAlias() {
+        IdentifierValue alias = null;
+        DatabaseType databaseType = null;
+        alias = new IdentifierValue("Data", QuoteCharacter.NONE);
+        assertThat(ProjectionUtils.getColumnLabelFromAlias(alias, databaseType), is(alias.getValue()));
+        alias = new IdentifierValue("Data");
+        databaseType = new PostgreSQLDatabaseType();
```
Comment (strongduanmu) [inline infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java:41]: Same suggestion with databaseType.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java (hunk 4 above)
Comment (pandalee99) [reply infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/util/ProjectionUtilsTest.java:41]: thanks,i can

**Comment by pandalee99** [comment]
CI / CI have problem
> Error:   package org.apache.shardingsphere.infra.util.quote does not exist
I didn't change this class.  but he tell me have problem.
so,what should I do.

**Comment by strongduanmu** [comment]
You can merge master branch code.

**Comment by pandalee99** [comment]
i am done .maybe,can be approve

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**